test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -18,18 +18,43 @@
 #ifndef IROHA_AMETSUCHI_FIXTURE_HPP
 #define IROHA_AMETSUCHI_FIXTURE_HPP
 
+#include <gtest/gtest.h>
+#include <cpp_redis/cpp_redis>
+#include <pqxx/pqxx>
+
 #include "common/files.hpp"
+#include "crypto/crypto.hpp"
 #include "logger/logger.hpp"
 #include "main/config/config.hpp"
+#include "util/string.hpp"
+#include "main/config/config.hpp"
 
-#include <gtest/gtest.h>
-#include <cpp_redis/cpp_redis>
-#include <pqxx/pqxx>
-#include "config_fake.hpp"
+using namespace std::literals::string_literals;
+using iroha::string::util::from_string;
 
+template <typename T>
+T parseEnv(const char *name, T default_) {
```

Comment:
add comments plz
looks like a useful function, maybe move it out from the tests?

Reply:
Not now. Now we don't need to parse env variables.